src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -191,7 +213,7 @@ class AddressValidationModal extends React.Component {
             : 'Edit home address'
         }
         id="address-validation-warning"
-        onClose={closeModal}
+        onClose={this.props.closeModal}
```

Comment:
Any reason to update this and break the current pattern of destructuring props at the top of the component?

Reply:
@Clark-W-Griswold This avoids a linter error that closeModal is defined in the upper scope - its either this or rename it on the import.

Reply:
Feels wrong to introduce this because of a linting error. :(